ftw/showroom/js/src/showroom.js

```diff
@@ -173,6 +182,11 @@ module.exports = function Showroom(items = [], options) {
     checkArrows();
   }
 
+  function refresh() {
+    options.references = Array.prototype.slice.call(document.querySelectorAll(".showroom-reference"));
```

Comment:
i thought the `showroom-item` class is already defined by the client, so for the sake of consistency we could provide a configuration option. but you're right it is optional 😉 .